uyuni-tools/reactivate_proxy_clients.py

```diff
@@ -0,0 +1,82 @@
+#!/bin/python3
+
+import argparse
+import getpass
+import salt.client
+import sys
+from xmlrpc.client import ServerProxy
+
+parser = argparse.ArgumentParser(description = "Reactivate all online clients attached to the specified proxy (determined by salt clients 'master' grain)")
+parser.add_argument("--dryrun", default=False, action="store_true", help = "Show the actions, but do not do anything")
+parser.add_argument("--limit", default=None, help = "Limit the number of modified salt clients (default unlimited)", type=int)
+parser.add_argument("--host", help = "SUSE Manager hostname", default = "localhost")
+parser.add_argument("--user", help = "SUSE Manager username for login. If not provided, will be asked for")
+parser.add_argument("--password", help = "SUSE Manager password for login. If not provided, will be asked for")
+parser.add_argument("proxy_fqdn", help = "FQDN or proxy which clients are to be reactivated")
+
+args = parser.parse_args()
+
+MANAGER_URL = "http://{}/rpc/api".format(args.host)
```

Comment:
Shouldn't this use or default to `https`?

Reply:
You are right. Changed to https. This breaks host default value `localhost` so I will remove it.